.travis.yml

```diff
@@ -9,4 +9,11 @@ node_js:
   - 8
   - 10
   - 12
-  - 14
\ No newline at end of file
+  - 14
+
```

Comment:
Could add Node.js 15 to the matrix as that should already provide npm v7.

Reply:
I prefer testing on stable node